contracts/root/lib/ExitTxValidator.sol

```diff
@@ -0,0 +1,65 @@
+pragma solidity ^0.5.2;
+
+import { BytesLib } from "../../common/lib/BytesLib.sol";
+import { Common } from "../../common/lib/Common.sol";
+import { RLPEncode } from "../../common/lib/RLPEncode.sol";
+import { RLPReader } from "solidity-rlp/contracts/RLPReader.sol";
+
+library ExitTxValidator {
+  using RLPReader for bytes;
+  using RLPReader for RLPReader.RLPItem;
+  bytes constant public networkId = "\x0d";
+
+  // 0x2e1a7d4d = keccak256('withdraw(uint256)').slice(0, 4)
+  bytes4 constant WITHDRAW_FUNC_SIG = 0x2e1a7d4d;
+
+  /**
+   * @notice Process the transaction to start a MoreVP style exit from
+   * @param exitTx Signed exit transaction
+   * @param exitor Need I say more?
+   */
+  function processExitTx(bytes memory exitTx, address childToken, address exitor)
+    public
+    view
+    returns(uint256 exitAmountOrTokenId, bool burnt)
+  {
+    RLPReader.RLPItem[] memory txList = exitTx.toRlpItem().toList();
+    require(txList.length == 9, "MALFORMED_WITHDRAW_TX");
+    require(exitor == getAddressFromTx(txList), "TRANSACTION_SENDER_MISMATCH");
+    require(
+      childToken == RLPReader.toAddress(txList[3]), // corresponds to "to" field in tx)
+      "Reference and exit tx do not correspond to the same token"
+    );
+    bytes memory txData = RLPReader.toBytes(txList[5]);
+    bytes4 funcSig = BytesLib.toBytes4(BytesLib.slice(txData, 0, 4));
+    if (funcSig == WITHDRAW_FUNC_SIG) {
+      require(txData.length == 36, "Invalid tx"); // 4 bytes for funcSig and a single bytes32 parameter
+      exitAmountOrTokenId = BytesLib.toUint(txData, 4);
+      burnt = true;
```

Comment:
Make sure we don't set burnt true for every transaction.

Reply:
For sure :)